tasty-plutus/src/Test/Tasty/Plutus/Context.hs

```diff
@@ -25,6 +25,7 @@ module Test.Tasty.Plutus.Context (
 
   -- ** Building contexts
   ExternalType (..),
+  ValueType (..),
```

Comment:
If we're going to expose `ValueType`, we should document it with a Haddock (complete with `since`) in the module it comes from. We also need to note this in the changelog.

Reply:
Noted. This is draft so far, I am exposing things as I need them and will fix up and push when ready.

Reply:
Ah, I didn't see that, my bad!

Reply:
https://github.com/Liqwid-Labs/plutus-extra/blob/d1e612bfac56d8e739b8911dacec2e8e87e1713d/tasty-plutus/src/Test/Tasty/Plutus/Internal/Context.hs#L115

`ValueType` already haddocked, with wrong `since`. Maybe it was intended to be exported?

Reply:
Strange. Well, if it's Haddocked, it'll be fine.